linux_os/guide/system/accounts/accounts-banners/banner_etc_issue/ansible/shared.yml

```diff
@@ -5,13 +5,7 @@
 # disruption = medium
 {{{ ansible_instantiate_variables("login_banner_text") }}}
 
-- name: "{{{ rule_title }}} - remove incorrect banner"
-  file:
-    state: absent
-    path: /etc/issue
-
-- name: "{{{ rule_title }}} - add correct banner"
-  lineinfile:
+- name: "{{{ rule_title }}}- ensure correct banner"
```

Comment:
I have just one nitpick, add a space before the dash, `-`.

Reply:
Done. Thanks for catch! : )